src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -81,6 +80,34 @@ describe('InfiniteScroll', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('mixed items', () => {
+    const tempItems = [
```

Comment:
This is a good start for the types of varied content we want to be able to support.

As it currently stands, we have 5 items here which would most likely fit on a single results page. With InfiniteScroll, users are presumably reaching for the component because they are likely presenting many pages of results.

How can we programmatically take what is here and populate a larger array which would span multiple pages of results? Loop through, splicing string lengths, vary numbers by iteration, vary strings using template literals and iterations, ... ?

Goal is to generate an array that spans multiple pages of results, as well as have enough variation so that resulting pageheights will vary while the items.length per page is consistent.

Let me know if a quick call would help to clarify, or if this is enough direction on which to iterate.

Reply:
Made some changes in the most recent push. Let me know what you think

Reply:
Nice. Thank you. Added a few more refining comments, then should be pretty close.